pkg/sdn/registry/clusternetwork/strategy.go

```diff
@@ -50,11 +49,7 @@ func (sdnStrategy) AllowCreateOnUpdate() bool {
 
 // ValidateUpdate is the default update validation for a ClusterNetwork
 func (sdnStrategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) fielderrors.ValidationErrorList {
-	result := fielderrors.ValidationErrorList{}
-	if obj.(*api.ClusterNetwork).Network != old.(*api.ClusterNetwork).Network {
-		result = append(result, fielderrors.NewFieldInvalid("Network", obj.(*api.ClusterNetwork), "cannot change the cluster's network CIDR midflight."))
-	}
-	return result
+	return validation.ValidateClusterNetworkUpdate(obj.(*api.ClusterNetwork), old.(*api.ClusterNetwork))
```

Comment:
mismatched order of obj,old. can this function go away and the signature of ValidateClusterNetworkUpdate be made compatible with RESTUpdateStrategy